3-async-await/solution.js

```diff
@@ -20,59 +20,52 @@ Example:
 9. as extra challenge: add Promise.race() and Promise.any(), and try to get the idea of what happens
 */
 
-const { log } = require("console");
 const prices = require("./prices");
 const products = require("./products");
 
-function solution() {
+async function solution() {
     // YOUR SOLUTION GOES HERE
 
     // You generate your id value here
     const id = parseInt(Date.now().toString().slice(-2));
 
     // You use Promise.all() here
-    const getProducts = async () => {
-        try {
-            const [product, price] = await Promise.all([
-                products(id),
-                prices(id),
-            ]);
-
-            console.log({ id: id, product: product, price: price });
-        } catch (error) {
-            console.log(error.message);
-        }
-    };
 
-    // You use Promise.allSettled() here
+    try {
+        const [product, price] = await Promise.all([products(id), prices(id)]);
 
-    const getProducts2 = async () => {
-        const results = await Promise.allSettled([
-            products(id), // {status, value}
-            prices(id), // {status, value}
-        ]);
+        console.log({ id: id, product: product, price: price });
```

Comment:
Ups, looks like I miss one. You can use the [shorthand property](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#property_definitions) here